tests/integration_tests/test_botorch.py

```diff
@@ -410,3 +410,22 @@ def test_call_after_trial_of_independent_sampler() -> None:
     ) as mock_object:
         study.optimize(lambda _: 1.0, n_trials=1)
         assert mock_object.call_count == 1
+
+
+@pytest.mark.parametrize("device", [None, torch.device("cpu"), torch.device("cuda:0")])
+def test_device_argument(device: Optional[torch.device]) -> None:
+
+    sampler = BoTorchSampler(device=device)
+    if not torch.cuda.is_available() and sampler._device.type == "cuda":
+        pytest.skip(reason="GPU is unavailable.")
```

Comment:
The result when GPU is available looks fine too: https://colab.research.google.com/drive/1PMfHH2ZlZ_FL1aFez5V56XzUmAFM2eil#scrollTo=3_b1lgABqyyw&line=1&uniqifier=1